boards/silabs/dev_kits/xg24_ek2703a/xg24_ek2703a_defconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 Silicon Labs
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ARM_MPU=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_GPIO=y
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_REGULATOR=y
+CONFIG_SOC_GECKO_EMU_DCDC=y
```

Comment:
Like the oscillator options in the other defconfig file, these DCDC options don't do anything either, configuration is now done in device tree.

Reply:
Deferred to a subsequent cleanup PR with the other defconfig file.